src/pages/[platform]/build-a-backend/storage/download-files/index.mdx

```diff
@@ -74,6 +74,8 @@ const linkToStorageFile = await getUrl({
     validateObjectExistence?: false,  // defaults to false
     expiresIn?: 20 // validity of the URL, in seconds. defaults to 900 (15 minutes) and maxes at 3600 (1 hour)
     useAccelerateEndpoint: true; // Whether to use accelerate endpoint.
+    bucket?: "userDefinedFriendlyBucketName" // Specify a target bucket using assigned name or full bucket name and region
```

Comment:
```suggestion
    bucket?: "userDefinedFriendlyBucketName" // Specify a target bucket using assigned name provided in Amplify backend
```

Reply:
We should be consistent with the language we use to refer to each option because it is kind of confusing. If we call the string option "friendly name" in the options on each page we should refer to it the same way here.

Reply:
It will be important to distinguish somewhere that the friendly name is the one provided by the customer and the bucket name is the actual generated bucket name that displays in the S3 console.